### PR TITLE
[ADD] Referrals: new rewards doc

### PR DESCRIPTION
**Pull request by larm-odoo**
Original Referrals doc too long, breaking off the rewards section into its own doc.

### CODE REVIEW

**Comment by larm-odoo** [comment]
Hi @brse-odoo, this is ready for a first-round review

**Review by brse-odoo — APPROVED** [review]
@larm-odoo I've finished my review of this PR, great job! I'm approving with optional suggestions for you to accept/reject as you see fit. Let me know if you have any questions, thanks!

File: content/applications/hr/referrals/rewards.rst
Comment (larm-odoo) [reply content/applications/hr/referrals/rewards.rst:56]: I think GUI- I keep getting dinged on that when it's a dashboard vs the real name, so I will use GUI since I feel that's what it ususally gets changed to =D

**Comment by larm-odoo** [comment]
Hi @ksc-odoo - this is ready for a final review.

File: content/applications/hr/referrals.rst
Comment (ksc-odoo) [inline content/applications/hr/referrals.rst:15]:
```suggestion
the :doc:`rewards <referrals/rewards>`; everything else is pre-configured when Odoo *Referrals*
```
no need for a space separating the end of the doc link and the semi-colon 👍

File: content/applications/hr/referrals.rst
Comment (ksc-odoo) [inline content/applications/hr/referrals.rst:14]:
```suggestion
The only configurations needed for the *Referrals* application *after* it has been installed, are related to
```
SUGGESTION to clean up the wording/pacing of the sentence a bit

File: content/applications/hr/referrals/rewards.rst
Comment (ksc-odoo) [inline content/applications/hr/referrals/rewards.rst:18]:
```suggestion
Rewards are the only configurations needed when setting up the *Referrals* application.

Only users with :guilabel:`Administrator` rights for the *Recruitment* application can create or
modify rewards.

To add rewards, navigate to :menuselection:`Referrals app --> Configuration --> Rewards`. Click
:guilabel:`New`, and a reward form loads. Enter the following information on the form:
```
SUGGESTION to break up this otherwise "lengthy" paragraph to make it a bit more digestible to the reader.

File: content/applications/hr/referrals/rewards.rst
Comment (ksc-odoo) [inline content/applications/hr/referrals/rewards.rst:24]: Just to make sure I'm understanding the explanation... does this mean if a reward should be used for several companies, there needs to a specific (separate) reward form for each company it will be applied to? If so, perhaps, it would be beneficial to consider slightly rewording this explanation to clarify that -- at the moment, it's a bit unclear how "each company needs to have a record of the reward..."

Just something to consider during your revisions...

File: content/applications/hr/referrals/rewards.rst
Comment (ksc-odoo) [inline content/applications/hr/referrals/rewards.rst:28]:
```suggestion
- :guilabel:`Gift Responsible`: using the drop-down menu, select the person responsible for
  procuring and delivering the reward to the recipient. This person is alerted when the reward is
  bought in the *Referrals* application, so they know when to deliver the reward to the recipient.
```
OPTIONAL reworking of the sentence structure to enhance the flow a bit

File: content/applications/hr/referrals/rewards.rst
Comment (ksc-odoo) [inline content/applications/hr/referrals/rewards.rst:29]:
```suggestion
- :guilabel:`Description` tab: type in the description for the reward. This is visible on the reward
```
just to clarify that it's a tab, not a field...

File: content/applications/hr/referrals/rewards.rst
Comment (ksc-odoo) [inline content/applications/hr/referrals/rewards.rst:37]: couple things here...

...and these are purely SUGGESTIONS...

- you may want to consider placing this explanation *before* the 'Description' tab, as it appears alongside the other fields at the top of the form
- perhaps, instead of 'image square' some more descriptive language could be used here to visually describe what it looks like (camera icon and plus sign icon, or something along those lines)
- While I fully appreciate the attention to detail here explaining how to upload an image, there may be a more succinct way to explaining it. For instance, I'm sure it's necessary to explain the appearance of two icons after its uploaded, and what they do...we may be able to assume the reader/user would understand what would happen if they clicked the 'trash can' icon, as that is, usually, a pretty universal icon...

Again, these are just SUGGESTIONS to consider during your revision 👍

File: content/applications/hr/referrals/rewards.rst
Comment (ksc-odoo) [inline content/applications/hr/referrals/rewards.rst:40]: SUGGESTION: I think this information may be best suited *at the end* of field explanation. Possibly with a "This is a **required** field" type of sentence.

File: content/applications/hr/referrals/rewards.rst
Comment (ksc-odoo) [inline content/applications/hr/referrals/rewards.rst:48]:
```suggestion
   the default cost is listed as zero, which would list the reward as free in the reward shop.
```

File: content/applications/hr/referrals/rewards.rst
Comment (ksc-odoo) [inline content/applications/hr/referrals/rewards.rst:49]:
```suggestion
   If a photo is not selected, a placeholder icon is displayed on the rewards page.
```

File: content/applications/hr/referrals/rewards.rst
Comment (ksc-odoo) [inline content/applications/hr/referrals/rewards.rst:58]:
```suggestion
In order to redeem a reward, points must be earned. These points can then be used to purchase a
reward.

To earn points for a reward, click the :guilabel:`Rewards` button on the main :guilabel:`Referrals`
dashboard. All the configured rewards are listed in individual reward cards.

The required point amount needed to purchase a reward is listed in the top-right corner of the card.
```
SUGGESTION: just some added clarification and broke up the paragraph a touch.

File: content/applications/hr/referrals/rewards.rst
Comment (ksc-odoo) [inline content/applications/hr/referrals/rewards.rst:62]:
```suggestion
reward card displays :guilabel:`You need another (x) points to buy this`, instead of a :icon:`fa-shopping-basket` :guilabel:`Buy` button.
```
added necessary GUI/icon to the Buy button mention at the end of the sentence

SUGGESTION: comma for enhanced sentence clarity

File: content/applications/hr/referrals/rewards.rst
Comment (ksc-odoo) [inline content/applications/hr/referrals/rewards.rst:66]:
```suggestion
reward. Click :guilabel:`OK` to purchase the item, or :guilabel:`Cancel` to close the window, and
```
SUGGESTION: comma for enhanced sentence clarity

**Review by ksc-odoo — APPROVED** [review]
Hey, @larm-odoo -- just finished my Final Review. Great work. Only a handful of suggestions/comments that require your attention. Approving now. Once the necessary adjustments have been implemented, feel free to tag this for tech review. Thanks! 👍

File: content/applications/hr/referrals/rewards.rst
Comment (larm-odoo) [reply content/applications/hr/referrals/rewards.rst:24]: Ahh, OK I see that. But yes, so if there are 3 companies, and they each have a $25 Amazon gift card reward, you will need to create 3 separate $25 Amazon Gift Card Entries, one for each company. I will reword!

**Comment by larm-odoo** [comment]
Hi @samueljlieber - this is ready for a tech review whenever =)

File: content/applications/hr/referrals.rst
Comment (samueljlieber) [inline content/applications/hr/referrals.rst:409]: Add bullets for multiple seealso items :)
```suggestion
   - :doc:`referrals/share_jobs`
   - :doc:`referrals/rewards`
```

**Review by samueljlieber — APPROVED** [review]
Hi @larm-odoo, nice job on this PR to move the Rewards content to it's own doc. I think this change makes sense. Your improvements look good to me, I just caught one quick fix before merge. Approving and delegating 👍
.....
@robodoo delegate=larm-odoo

File: content/applications/hr/referrals.rst
Comment (larm-odoo) [reply content/applications/hr/referrals.rst:409]: OMG I was used to removing the bullets when there was only one doc, I totally forgot!

**Comment by larm-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
